### PR TITLE
PR: Test icons for naming changes in QtAwesome

### PR DESCRIPTION
**Pull request by dalthviz**
## Description of Changes

* [ ] Wrote at least one-line docstrings (for any new functions)
* [x] Added unit test(s) covering the changes (if testable)

* [ ] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))

Use QtAwesome master for tests. Add test for the mapping of icons 

### Issue(s) Resolved

Related to #13858 
Related to #13875

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: dalthviz

### CODE REVIEW

**Comment by dalthviz** [comment]
Note: Depends on spyder-ide/qtawesome#155

File: .github/scripts/install.sh [hunk 1]
```diff
@@ -38,8 +38,9 @@ else
     # Install qtconsole from Github
     pip install git+https://github.com/jupyter/qtconsole.git
 
-    # Pin QtAwesome to 0.7.3 until we can update to 1.0.0
-    pip install qtawesome==0.7.3
+    # Install QtAwesome from Github
+    # pip install git+https://github.com/spyder-ide/qtawesome.git
+    pip install git+https://github.com/dalthviz/qtawesome.git@mdi_remap
```
Comment (ccordoba12) [inline .github/scripts/install.sh:43]: Now that https://github.com/spyder-ide/qtawesome/pull/155 is merged, you can change this to point to the main QtAwesome repo.

**Review by ccordoba12 — COMMENTED** [review]
Thanks @dalthviz for your prompt response to solve this problem! I left a minor comment, otherwise looks good.

**Review by ccordoba12 — APPROVED** [review]
Thanks @dalthviz!

**Pull request merged**